hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/execution/PrefixedLogger.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2008-2020, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.impl.execution;
+
+import com.hazelcast.logging.AbstractLogger;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.LogEvent;
+
+import java.util.logging.Level;
+import java.util.logging.LogRecord;
+
+public class PrefixedLogger extends AbstractLogger {
+
+    private static final String PATTERN = "%s -> %s";
```

Comment:
Maybe the `->` part should not be hardcoded here but by the user of this class. This class would then have a very simple contract and be in some `util` package.